core/src/main/java/org/dcache/nfs/v4/xdr/stateid4.java

```diff
@@ -21,45 +21,73 @@
 
 import java.io.IOException;
 import java.io.Serializable;
-import java.util.Arrays;
 
+import org.dcache.nfs.util.Opaque;
 import org.dcache.oncrpc4j.rpc.OncRpcException;
 import org.dcache.oncrpc4j.xdr.XdrAble;
 import org.dcache.oncrpc4j.xdr.XdrDecodingStream;
 import org.dcache.oncrpc4j.xdr.XdrEncodingStream;
 
-import com.google.common.io.BaseEncoding;
-
-public class stateid4 implements XdrAble, Serializable {
+public class stateid4 implements XdrAble, Serializable, Cloneable {
 
     static final long serialVersionUID = -6677150504723505919L;
 
-    public int seqid;
-    public byte[] other;
+    private int seqid;
+    private final Opaque other;
```

Comment:
NACK. `stateid4` is  Serializable. And this will break it. In particular for dCache.